src/Kontent.Ai.ModelGenerator/ArgHelpers.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Kontent.Ai.Delivery.Abstractions;
+using Kontent.Ai.Management.Configuration;
+using Kontent.Ai.ModelGenerator.Core.Configuration;
+
+namespace Kontent.Ai.ModelGenerator;
+
+public static class ArgHelpers
```

Comment:
We can do it internal right?

Reply:
https://github.com/kontent-ai/model-generator-net/blob/feature/154_improve_user_experience/src/Kontent.Ai.ModelGenerator/ArgHelpers.cs#L11